cg/store/models.py

```diff
@@ -1163,9 +1164,7 @@ class PacBioSampleSequencingMetrics(SampleRunMetrics):
     hifi_yield: Mapped[BigInt]
     hifi_mean_read_length: Mapped[BigInt]
     hifi_median_read_quality: Mapped[Str32]
-    percent_reads_passing_q30: Mapped[Num_6_2]
-    failed_reads: Mapped[BigInt]
-    failed_yield: Mapped[BigInt]
-    failed_mean_read_length: Mapped[BigInt]
+    polymerase_mean_read_length: Mapped[BigInt | None]
+    barcode: Mapped[Str32 | None]
```

Comment:
Do this change when we have the barcode metrics parsed and included to the database
```suggestion
    polymerase_mean_read_length: Mapped[BigInt]
    barcode: Mapped[Str32]
```

Reply:
fixed

Reply:
Depends on the default values we choose for them